content_schemas/formats/popular_links.jsonnet

```diff
@@ -0,0 +1,40 @@
+{
+  document_type: "popular_links",
+  base_path: "optional",
+  routes: "optional",
+  rendering_app: "optional",
+  title: "optional",
+  redirects: "optional",
+  definitions: {
+    details: {
+      type: "object",
+      additionalProperties: false,
+      properties: {
+        links_details: {
```

Comment:
We think there could be a better name for this field and we welcome any feedback on improving this name, the use of word link is very common inside publishing api and hence the hesitance.